### ISSUE TITLE
Jetpack Search: Make it easier to surface content written in both UK or US English variations

### ISSUE THREAD

**Issue by inaikem** [Automattic/wp-calypso#57589]
### What

Surface search results with both UK and US English variations of the search terms in Jetpack search.

### Why

Websites and P2s with globally distributed authors/contributors will have variations of both UK and US English spellings being used and searching for one version does not yield results in the other. This can lead to content and context being missed. 

On a site where one variation is used exclusively (eg https://wordpress.com/support), searching for a UK variation yields 0 results but suggests the US version. Example: https://wordpress.com/support?s=customise.

On a site where only _both_ variations are used interchangeably, different results appear depending on the variation searched for. Other variations are not suggested/included. For example, search `prioritize` vs `prioritise` on a P2 with Jetpack Instant Search enabled.

See:

- cLjiI-p2?s=prioritise
- cLjiI-p2?s=prioritize

### How

Add UK/US and other EN roots, derivatives and variations to Jetpack search and include matched results in search.

**Comment by trakos** [comment]
@gibrown do you think that we could close this issue in favour of epic https://github.com/Automattic/jetpack-roadmap/issues/2144? I feel like that epic covers the best solution for this.

Alternatively, we could expand fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Scyhtvaf%2Srynfgvpfrnepu%2Svaqrkvat%2Sjcrf%2Qyvo%2Sfep%2Spbzzba%2Spynff.jcrf%2Qnanylmre%2Qohvyqre%2Qi2.cuc%3Se%3Qp61p75n9%231358-og to introduce a synonym filter into our mappings for British spellings. See [this stackoverflow comment](https://stackoverflow.com/a/32023292) for an example. However, that would require a rebuild, and would be a cat-and-mouse game to cover all synonyms and possible spellings.

**Comment by gibrown** [comment]
Good idea adding it to that roadmap issue. I suspect that will fix it, but I'm not sure we want to assume it will. https://github.com/Automattic/jetpack/issues/8895#issuecomment-446697250 also has some solutions that involve synonyms, so added that issue as well.

I think we should keep them open as reminders to verify that we have actually fixed the issues, but I lowered the priority on both of them to Low for now.
